src/command_modules/azure-cli-resource/HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+2.1.5
++++++
+* `policy definition create`: Fixed the incorrect parameter definition example.
+* `policy definition create`: Replaced the the use of `-n` alias with full `--name` parameter in the example.
```

Comment:
Can you remove the redundant `the` here? Also, I suggest make it simple and only keep one entry like `policy definition create: fix examples`